ichnaea/tests/base.py

```diff
@@ -76,6 +76,7 @@
     },
     'locate:fallback': {
         'url': 'http://127.0.0.1:9/?api',
+        'ratelimit': 10,
```

Comment:
Can you make this a string here? At this point there's no magic yet which would have converted it from a string to an int.